### PR TITLE
Bump devpack to version 3.1.7

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,7 +109,7 @@
     "ci": "./build.sh",
     "clean-svg": "svgo --pretty --config=.svgo.yml -f icons -o icons",
     "doc": "parallelshell 'npm run doc:html' 'npm run doc:js' 'npm run doc:css'",
-    "doc:css": "cssnext index.css > bundle.css && cssnext example.css >> bundle.css",
+    "doc:css": "cat index.css example.css | cssnext --sourcemap > bundle.css",
```
Comment (fabiosantoscode) [inline package.json]: cool, this

File: package.json [hunk 2]
```diff
@@ -15,16 +15,18 @@
   "main": "index.js",
   "files": [
     "*.js",
-    "example.es6",
-    "index.es6",
```
Comment (fabiosantoscode) [inline package.json]: nope, these *must* be exported because component gallery imports them.

File: package.json (hunk 1 above)
Comment (fampinheiro) [inline package.json]: Also liked it :+1:

**Comment by fabiosantoscode** [comment]
Alright, mergin'

**Pull request merged**